### PR TITLE
Release 2.3.1

### PR DESCRIPTION
**Pull request by sj23patel**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Config [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# If the current branch does not yet exist in Brazil, you can create and push
```
Comment (seiyako) [inline Config:1]: Config file should be removed

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-streams",
-  "version": "2.3.0",
```
Comment (seiyako) [inline package.json:3]: version has to be 2.3.1

File: package.json [hunk 3]
```diff
@@ -27,15 +27,15 @@
   "scripts": {
     "release": "npm run build-streams && npm run test-mocha",
     "build-streams": "webpack --config ./webpack/connect-streams.config.js && webpack --config ./webpack/connect-streams-min.config.js",
-    "test-mocha": "nyc mocha \"test/unit/**/*.spec.js\" --exit",
+    "test-mocha": "nyc mocha \"test/unit/**/*.spec.js\" --ignore \"test/unit/drCoordinator/**\" --exit",
```
Comment (seiyako) [inline package.json:30]: remove dr related script

File: src/worker.js [hunk 4]
```diff
@@ -720,6 +720,11 @@
       this.agent.configuration.routingProfile.routingProfileId =
         this.agent.configuration.routingProfile.routingProfileARN;
       
+      if (this.suppress) {
```
Comment (seiyako) [inline src/worker.js:723]: dr related code should be removed

**Review by seiyako — APPROVED** [review]

**Review by kvs6rj — APPROVED** [review]

**Review by ShanshanXu2021 — APPROVED** [review]

**Pull request merged**